ui/tract-renderer/build.gradle

```diff
@@ -65,9 +65,12 @@ dependencies {
     implementation "com.tinder.statemachine:statemachine:${deps.tinderStateMachine}"
     implementation "jp.wasabeef:picasso-transformations:${deps.picassoTransforms}"
 
+    compileOnly "com.squareup.inject:assisted-inject-annotations-dagger2:${deps.assistedInject}"
```

Comment:
using square assisted inject is temporary until https://github.com/google/dagger/issues/2278 is released